packages/wrangler/src/deployment-bundle/esbuild-plugins/nodejs-compat.ts

```diff
@@ -35,18 +38,15 @@ export const nodejsCompatPlugin: (silenceWarnings: boolean) => Plugin = (
 				const result = await pluginBuild.resolve(path, {
 					kind,
 					resolveDir,
-					importer: opts.importer,
+					importer,
 				});
 
 				if (result.errors.length > 0) {
 					// esbuild couldn't resolve the package
 					// We should warn the user, but not fail the build
-
-					let pathWarnedPackaged = warnedPackaged.get(path);
-					if (pathWarnedPackaged === undefined) {
-						warnedPackaged.set(path, (pathWarnedPackaged = []));
-					}
-					pathWarnedPackaged.push(opts.importer);
+					const pathWarnedPackaged = warnedPackaged.get(path) ?? [];
```

Comment:
I think these variable names were typos originally...

```suggestion
					const pathWarnedPackages = warnedPackages.get(path) ?? [];
```